pkg/kubelet/kubelet.go

```diff
@@ -927,10 +927,11 @@ func makeMounts(container *api.Container, podVolumes kubecontainer.VolumeMap) (m
 			continue
 		}
 		mounts = append(mounts, kubecontainer.Mount{
```

Comment:
This is the right place to calculate whether this is the first container that mounts the volume and drive `SELinuxRelabel` based on that.